styleguide/_themes/derek/scss/patterns/grid/ias.scss

```diff
@@ -0,0 +1,115 @@
+.ias-container {
+  @include container-fixed;
+  max-width: 960px;
+  padding: $standard-padding;
+
+  @media (min-width: $screen-sm-min) {
+    width: $container-sm;
+  }
+  @media (min-width: $screen-md-min) {
+    width: $container-md;
+  }
+  @media (min-width: $screen-lg-min) {
+    width: $container-lg;
+  }
+}
+
+.ias-row {
+  @include make-row;
+}
+
+@media only screen and (min-width: $screen-sm-min) {
+  .ias-row {
+    @include flex-box;
+  }
+}
+
+.ias-col-full {
+  @include make-md-column(12);
+  @include flex-box;
+}
+
+.ias-col-half {
+  @include make-sm-column(6);
+  @include flex-box;
+}
+
+.ias-col-third {
+  @include make-xs-column(12);
+  @include make-sm-column(4);
+  @include flex-box;
+}
+
+.ias5-col-push {
+  @include make-sm-column-push(2);
+}
+
+.ias5-col {
+  @include make-sm-column(4);
+  @include flex-box;
+}
+
+.ias-title {
+  text-align: center;
+}
+
+.iasWidget {
+  padding: 20px 10px 0;
+  width: $full-width;
+}
+
+.iasWidget-body {
+  color: $darkest-gray;
+  font-family: $copy;
+  font-weight: 300;
+  line-height: 1.5em;
+  margin: 0 0 1em;
+  text-align: center;
+}
+
+.iasWidget-col-full {
+  @include make-md-column(12);
+}
+
+.iasWidget-cta {
+  color: $rackspace-red;
+  font-family: $copy;
+  font-weight: 600;
+  text-align: center;
+}
+
+.iasWidget-header {
+  color: $darkest-gray;
+  font-family: $heading;
+  font-size: 1.25em;
+  font-weight: 400;
+  line-height: 1.25em;
+  margin: 10px 0;
+  text-align: center;
```

Comment:
the only thing that should be non default for the h3 is the centering. We shouldnt define all of the rest because when the h3 changes, we're going to want this to change as well. If we switch to a new font (a reality very soon) then we'll need the font-size, weight and line height to change as well. If we define this so heavily it will complicate that process quite a bit.

Also, the font weight is incorrect. This needs to be 300 or removed. (I recommend removed)